### DISCUSSION TITLE
Beatmap length/play time display in mod select

### DISCUSSION THREAD

**Discussion by Tenexxt** [ppy/osu#29834] [Q&A]
I personally think that drain time and/or total lenght of the map should be displayed in the mod select menu.
I find the lenght of a map an important enough stat to be shown in the mod select, as it would be very helpfull with mods that affect time, since you could know how long is the map directly instead of having to guess because the game only gives you the NM value. 
Tho i have to admit, that im not sure how it would work for WU and WD, and especially AS, but _In My Opinion_ i think giving it to the main 4 (DC, HT, DT, NC) would be fine, but let me know what you think.

**Comment by Qunas101** [comment]
Bump. The only way right now to see the modified map length is to peek under the mod select overlay, which only works on song select. It is impossible to know beatmap length in daily challenge and playlists

**Reply by peppy** [reply]
![osu! 2025-04-22 at 06 52 51](https://github.com/user-attachments/assets/6d5ec2c5-22ea-4a9c-889f-ea738b9c041c)

is this not what you're after?

**Reply by Qunas101** [reply]
Sure, maybe 'impossible' was a poor choice of words. But still, that only covers base speed. Making players do math in their head is not the best UX I think.
You may say it's not important to know the EXACT timing, and maybe you'd be right, but I think it's nice to have, equally as nice as knowing modified OD stats
